R-CMD-check action failing

I just merged #205 since it was minor and might avoid errors during JOSS reviews and the R-CMD-check action failed. It doesn't appear to have anything to do with the change in #205, it's showing external dependencies issues in the vignette building.

```sh
 ── R CMD build ─────────────────────────────────────────────────────────────────
* checking for file ‘.../DESCRIPTION’ ... OK
* preparing ‘portalcasting’:
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
* creating vignettes ... ERROR
Error: --- re-building ‘adding_model_and_data.Rmd’ using rmarkdown
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Error: Error: processing vignette 'adding_model_and_data.Rmd' failed with diagnostics:
unable to load shared object '/home/runner/work/_temp/Library/stringi/libs/stringi.so':
  libicui18n.so.60: cannot open shared object file: No such file or directory
Error: Error: tangling vignette 'adding_model_and_data.Rmd' failed with diagnostics:
unable to load shared object '/home/runner/work/_temp/Library/stringi/libs/stringi.so':
  libicui18n.so.60: cannot open shared object file: No such file or directory
--- failed re-building ‘adding_model_and_data.Rmd’

--- re-building ‘codebase.Rmd’ using rmarkdown
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Error: Error: processing vignette 'codebase.Rmd' failed with diagnostics:
unable to load shared object '/home/runner/work/_temp/Library/stringi/libs/stringi.so':
  libicui18n.so.60: cannot open shared object file: No such file or directory
Error: Error: tangling vignette 'codebase.Rmd' failed with diagnostics:
unable to load shared object '/home/runner/work/_temp/Library/stringi/libs/stringi.so':
  libicui18n.so.60: cannot open shared object file: No such file or directory
--- failed re-building ‘codebase.Rmd’

--- re-building ‘current_models.Rmd’ using rmarkdown
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Error: Error: processing vignette 'current_models.Rmd' failed with diagnostics:
unable to load shared object '/home/runner/work/_temp/Library/stringi/libs/stringi.so':
  libicui18n.so.60: cannot open shared object file: No such file or directory
Error: Error: tangling vignette 'current_models.Rmd' failed with diagnostics:
unable to load shared object '/home/runner/work/_temp/Library/stringi/libs/stringi.so':
  libicui18n.so.60: cannot open shared object file: No such file or directory
--- failed re-building ‘current_models.Rmd’

--- re-building ‘getting_started.Rmd’ using rmarkdown
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Error: Error: processing vignette 'getting_started.Rmd' failed with diagnostics:
unable to load shared object '/home/runner/work/_temp/Library/stringi/libs/stringi.so':
  libicui18n.so.60: cannot open shared object file: No such file or directory
Error: Error: tangling vignette 'getting_started.Rmd' failed with diagnostics:
unable to load shared object '/home/runner/work/_temp/Library/stringi/libs/stringi.so':
  libicui18n.so.60: cannot open shared object file: No such file or directory
--- failed re-building ‘getting_started.Rmd’

--- re-building ‘jags_models.Rmd’ using rmarkdown
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Error: Error: processing vignette 'jags_models.Rmd' failed with diagnostics:
unable to load shared object '/home/runner/work/_temp/Library/stringi/libs/stringi.so':
  libicui18n.so.60: cannot open shared object file: No such file or directory
Error: Error: tangling vignette 'jags_models.Rmd' failed with diagnostics:
unable to load shared object '/home/runner/work/_temp/Library/stringi/libs/stringi.so':
  libicui18n.so.60: cannot open shared object file: No such file or directory
--- failed re-building ‘jags_models.Rmd’

SUMMARY: processing the following files failed:
  ‘adding_model_and_data.Rmd’ ‘adding_model_and_data.Rmd’
  ‘codebase.Rmd’ ‘codebase.Rmd’ ‘current_models.Rmd’
  ‘current_models.Rmd’ ‘getting_started.Rmd’ ‘getting_started.Rmd’
  ‘jags_models.Rmd’ ‘jags_models.Rmd’

Error: Error: Vignette re-building failed.
Execution halted
Error: Error in proc$get_built_file() : Build process failed
Calls: <Anonymous> ... build_package -> with_envvar -> force -> <Anonymous>
Execution halted
Error: Process completed with exit code 1.
```

--- a/.github/workflows/check-release.yaml
+++ b/.github/workflows/check-release.yaml
@@ -12,10 +12,10 @@ name: R-CMD-check-release
 
 jobs:
   R-CMD-check:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
-      CRAN_REPO: https://packagemanager.rstudio.com/all/__linux__/bionic/latest
+      CRAN_REPO: https://packagemanager.rstudio.com/all/__linux__/focal/latest
     steps:
       - uses: actions/checkout@v2
       - uses: r-lib/actions/setup-r@v1

--- a/.github/workflows/pkgdown.yaml
+++ b/.github/workflows/pkgdown.yaml
@@ -11,10 +11,10 @@ name: pkgdown
 
 jobs:
   pkgdown:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
-      CRAN_REPO: https://packagemanager.rstudio.com/all/__linux__/bionic/latest
+      CRAN_REPO: https://packagemanager.rstudio.com/all/__linux__/focal/latest
     steps:
       - uses: actions/checkout@v2
 

--- a/.github/workflows/test-coverage.yaml
+++ b/.github/workflows/test-coverage.yaml
@@ -10,10 +10,10 @@ name: test-coverage
 
 jobs:
   test-coverage:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
-      CRAN_REPO: https://packagemanager.rstudio.com/all/__linux__/bionic/latest
+      CRAN_REPO: https://packagemanager.rstudio.com/all/__linux__/focal/latest
     steps:
       - uses: actions/checkout@v2
 

